env.go

```diff
@@ -138,7 +138,17 @@ func configure(opts []Options) []Options {
 func toMap(env []string) map[string]string {
 	r := map[string]string{}
 	for _, e := range env {
+		// On Windows, environment variables can start with '='. If so, Split at next character.
```

Comment:
maybe we should do this only if goos is windows?

Reply:
@caarlos0 done